src/html/tags/HtmlTagStrong.js

```diff
@@ -4,8 +4,8 @@ import { Text } from 'react-native';
 import styles from '../HtmlStyles';
 import renderHtmlChildren from '../renderHtmlChildren';
 
-export default ({ cascadingStyle, ...restProps }) => (
-  <Text style={[styles.b, cascadingStyle]}>
-    {renderHtmlChildren({ cascadingStyle, ...restProps })}
+export default ({ cascadingStyle, cascadingChildStyle, ...restProps }) => (
+  <Text style={[styles.b, cascadingChildStyle]}>
```

Comment:
I guess the previous code is not correct, as we wouldn't want it to be a `<Text />` here, but a `<View />`.